### DISCUSSION TITLE
Will the NDI plugin reroute my music over the Twitch VOD audio channel?

### DISCUSSION THREAD

**Discussion by magnuscodus** [DistroAV/DistroAV#697] [Q&A - Support] [answered]
Hello,

I tried to make my title as precise as I could, but I am better off explaining it here.
So I recently discovered the NDI plugin, and I have tried it on my last two streams. I seems to be working perfectly except one thing..
On my gaming PC I have managed to route my copyrighted music through the twitch VOD audio track ( for me it was the 6 audio track ) and till now everything worked perfectly fine, and my music was not to be heard on the VOD, only in my streams.
The thing is, that since I've started using NDI, I saw that my music is being heard on stream and thus large portions of the VOD are getting muted.

Do I just activate the Twitch VOD audio track ( I will probably use the 6th audio track as well on my encoding pc ) and it works , as it worked on the gaming pc, or do I have to come up with a more complex solution to reroute my music on the VOD and thus Twitch Soundtrack?

Thank you a lot in advance for your help!

PS: I probably explained my problem quite complicated, but I am happy to provide more information!

**Comment by Trouffman** [comment, marked as answer]
This is most likely an audio configuration and not related to the plugin. Ensure you follow the proper steps for the Twitch VOD configuration, making sure that your NDI source is not on that audio track nor in some filter.
